docs/index.adoc

```diff
@@ -8060,7 +8060,7 @@ class ListFiles : Runnable {
 ----
 
 === Subcommand Aliases
-Commands may optionally define an `aliases` attribute to provide alternate names for commands that will be recognized by the parser. Aliases are displayed in the default help output. For example:
+Commands may optionally define an `aliases` attribute to provide alternate names for subcommands that will be recognized by the parser. Aliases are displayed in the default help output. For example:
```

Comment:
Reading this again, I realized that maybe this "for commands" or "for subcommands" can be confusing. 
How about leaving it out altogether? So we get:

> ... provide alternate names that will be recognized by the parser.

Reply:
I agree, it's better this way